cluster/manifests/01-vertical-pod-autoscaler/rbac.yaml

```diff
@@ -0,0 +1,290 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:metrics-reader
+rules:
+  - apiGroups:
+      - "metrics.k8s.io"
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:vpa-actor
+rules:
+  - apiGroups:
+      - ""
```

Comment:
what does the empty apiGroup mean?

Reply:
Empty is for standard resources like pods. They have `apiVersion: v1` where as ingress for example has `apiVersion: extensions/v1beta1`. The part before the version is the `apiGroup`